flocker/node/agents/blockdevice.py

```diff
@@ -550,37 +550,34 @@ class MountBlockDevice(PClass):
 
     :ivar UUID dataset_id: The unique identifier of the dataset associated with
         the filesystem to mount.
-    :ivar unicode blockdevice_id: The unique identifier of the
-        ``IBlockDeviceAPI``-managed volume to be mounted.
+    :ivar FilePath device_path: The path of the block device to be mounted.
     :ivar FilePath mountpoint: The filesystem location at which to mount the
         volume's filesystem.  If this does not exist, it is created.
     """
-    dataset_id = field(type=UUID, mandatory=True)
-    blockdevice_id = field(type=unicode, mandatory=True)
+    device_path = field(type=FilePath, mandatory=True)
     mountpoint = field(type=FilePath, mandatory=True)
 
+    # Only for logging
+    dataset_id = field(type=UUID, mandatory=True)
```

Comment:
Hmhm. Presuming sufficient logging elsewhere then probably unnecessary... but I'm not sure we actually log the right info. I'd say leave it in and when we're done refactoring do another pass of logging improvements,